app/util.py

```diff
@@ -20,6 +20,7 @@
 from app.db import get_db, handle_skipped_award
 from app.exceptions import SkippedAwardError
 from app.i18n import _
+from app.models import ApplicationAction, ApplicationActionType
```

Comment:
We already have `from app import models`, so change the code to do `models.Application...`